integrationtests/utils.go

```diff
@@ -346,3 +346,113 @@ func volumeInit(volumeClient volume.Interface, t *testing.T) (*VirtualHardDisk,
 	}
 	return vhd, volumeID, vhdCleanup
 }
+
+const letterset = "abcdefghijklmnopqrstuvwxyz" + "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+func stringWithCharset(length int, charset string) string {
```

Comment:
These are utils not used by any other integration test, I'd limit the scope of these functions to be inside the SMB test

I think `volumeInit` was used in many API Groups so I guess that one is ok

Reply:
moved back in latest commit :)